test-butler-library/src/main/java/com/linkedin/android/testbutler/TestButler.java

```diff
@@ -158,6 +158,22 @@ public static void setWifiState(boolean enabled) {
     }
 
     /**
+     * Enable/disable the Wifi connection on the emulator
```

Comment:
Airplane Mode have some problems:

1. It is even more hacky then `TelephonyManager`  api. 
2. It also disables location services/Bluetooth - which sometimes may be critical to clients

Nevertheless, I think library should have API for both GSM data transfer disabling and  Toogling Airplane mode